docs/topics/eap.md

```diff
@@ -26,4 +26,33 @@ check [our instructions on how to configure your build to support this version](
 
 ## Build details
 
-_No preview versions are currently available._
\ No newline at end of file
+<table>
+    <tr>
+        <th>Build info</th>
+        <th>Build highlights</th>
+        <th>Recommended kotlinx library versions</th>
+    </tr>
+    <tr>
+        <td><strong>1.6.10-RC</strong>
+            <p>Released: <strong>December 7, 2021</strong></p>
+            <p><a href="https://github.com/JetBrains/kotlin/releases/tag/v1.6.10-RC" target="_blank">Release on GitHub</a></p>
+        </td>
+        <td>
+             <ul>
+                <li>Release candidate for a bug fix release for Kotlin 1.6.0.</li>
+            </ul>
```

Comment:
I wouldn't use the list for this single item.

Reply:
Also, there are two "for"s in a short sentence. Maybe "Release candidate for a bug fix release Kotlin 1.6.10."?

Reply:
Thanks, I like this